modules/system/applications.nix

```diff
@@ -37,11 +38,40 @@ in
         rm ~/Applications/'Nix Apps'
       fi
 
-      if [ ! -e '/Applications/Nix Apps' ] \
-         || ourLink '/Applications/Nix Apps'; then
-        ln -sfn ${cfg.build.applications}/Applications '/Applications/Nix Apps'
+      ourfolder () {
+        local marker
+        marker=$(xattr -p org.nix-darwin "$1" 2>/dev/null)
+        [ "$marker" = 'generated' ]
+      }
+
+      targetFolder='/Applications/Nix Apps'
+
+      # Clean up for links created at the old location in /Applications
```

Comment:
nit: Not the old location, right?

Reply:
Hm, that could be clearer. What do you think of 'Clean up old style symlink to nix store'

Reply:
Sounds fine to me 👍